src/app/main/main.component.ts

```diff
@@ -210,18 +215,24 @@ export class MainComponent implements OnInit, OnDestroy {
         });
 
       if (this._hasJobParams() && this._jobParamsValid()) {
+        this.dataFormGroup.patchValue({
+          sql: '/* Loading sql query from job... */',
+          projectID: this.project,
+          location: this.location
+        });
         this.dataService.getQueryFromJob(this.jobID, this.location, this.project).then((queryText) => {
           this.queryTextFromJob = queryText.sql;
```

Comment:
It doesn't look like `this.queryTextFromJob` is being used anywhere, can we just keep it a local variable?

Reply:
Yup. good catch.